services/client/src/selectors/index.js

```diff
@@ -5,3 +5,5 @@ export const getUpcomingEvents = state => state.events.upcomingEvents;
 export const isLoading = state => state.events.isLoading;
 
 export const hasErrors = state => state.events.hasErrors;
+
+export const hasMoreItems = state => state.events.hasMoreItems;
```

Comment:
Is this file duplicated?

Reply:
Can't see anywhere this file is referenced - does the app still work if you nuke this file?

Reply:
It does, should I delete it?

Reply:
yes please; nuke it :D